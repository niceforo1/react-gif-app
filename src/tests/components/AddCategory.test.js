import React from 'react';
import { shallow } from 'enzyme';
import { AddCategory } from '../../components/AddCategory';

describe('Test in <AddCategory/> component', () => {
  const setCategories = () => {};
  const wrapper = shallow(<AddCategory setCategories={setCategories} />);

  test('should be render <AddCategory/> ', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should be change the input text', () => {
    /*const input = wrapper.find('input');

    input.simulate('change');*/
  });
});
