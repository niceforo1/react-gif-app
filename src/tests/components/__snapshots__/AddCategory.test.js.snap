// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <AddCategory/> component should be render <AddCategory/>  1`] = `
<form
  onSubmit={[Function]}
>
  <h3>
    Add Category
  </h3>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
